### PR TITLE
Fix screen flicker issue

### LINKED ISSUE #6: Every page flickers after a few seconds
A few seconds after a page is loaded it flickers, just before the 'consent fingerprint' appears in the bottom left corner. See video:

https://github.com/hlxsites/sunstar/assets/226514/d9023a3e-04e3-4830-a1b9-3e6935671f01

### PR DESCRIPTION
**Pull request by bosschaert**
Fixes #6

Test URLs:
- Before: https://main--sunstar--hlxsites.hlx.page/
- After: https://issue-6--sunstar--hlxsites.hlx.page/

Applied suggested fix from @hparwani2 in https://github.com/hlxsites/sunstar/issues/6#issuecomment-1729271153

### CODE REVIEW

**Comment by bosschaert** [comment]
Note the visual differences reported are because the branch was created off main a while ago...

File: head.html [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+            if (a) {
+                var e = b.createElement("style");
+                e.id = c;
+                e.innerHTML = d;
+                a.appendChild(e)
+            }
+        }
+        )(b.getElementsByTagName("head")[0], "at-body-style", d);
+    }
+    )(window, document, "body {opacity: 1 !important}");
```
Comment (sdmcraft) [inline head.html:16]: I hope this plays well with other `opacity` settings. For e.g. if we set the `opacity` somewhere to `0` , you may want to check if this isn't inadvertently overriding that.

**Review by sdmcraft — COMMENTED** [review]
I added a comment that you may want to verify. If it turns that its not a concern, we should be good to merge.

File: head.html (hunk 1 above)
Comment (bosschaert) [reply head.html:16]: Doesn't it only apply this to elements with name `at-body-style`?

**Review by sdmcraft — APPROVED** [review]

**Comment by sdmcraft** [comment]
> Note the visual differences reported are because the branch was created off main a while ago...

you may want to merge `main` just to be sure that nothing changes visually.

File: head.html (hunk 1 above)
Comment (bosschaert) [reply head.html:16]: Maybe @hparwani2 can clarify this question, since the patch comes from him 😄

**Comment by bosschaert** [comment]
> you may want to merge `main` just to be sure that nothing changes visually.

Have done that now and the visual tests are not reporting any changes.

**Pull request merged**